app/scripts/modules/core/src/plugins/plugin.module.ts

```diff
@@ -12,8 +11,24 @@ module(PLUGINS_MODULE, ['ui.router']).config([
     // Tell the router to slow its roll
     $uiRouterProvider.urlService.deferIntercept();
 
-    // Grab all plugins that are defined in Spinnaker settings
-    SETTINGS.plugins.forEach(plugin => pluginRegistry.register(plugin));
+    // Grab all plugins that are defined in plugin-manifest
+    // The format for plugin-manifest would be:
+    //    const PLUGINS = [{'name':'myPlugin', 'version':'1.2.3', 'devUrl':'/plugins/index.js'}]
+    //    export { PLUGINS }
+    try {
+      const pluginManifestLocation = '/plugin-manifest.js';
+      const pluginModule = await import(/* webpackIgnore: true */ pluginManifestLocation);
+
+      if (!pluginModule || !pluginModule.PLUGINS) {
+        throw new Error(`Error loading plugins.`);
+      } else {
+        // @ts-ignore
+        pluginModule.PLUGINS.forEach(plugin => pluginRegistry.register(plugin));
+      }
+    } catch (error) {
+      console.error('No plugins found.');
```

Comment:
This error handling doesn't make much sense to me. 

1) It swallows the original error
2) It will log 'no plugins found' if `pluginRegistry.register()` fails, which is misleading

I'd prefer to see something like:
```suggestion
      console.error('Error registering plugin manifests');
      console.error(error);
```